cms/envs/common.py

```diff
@@ -1536,3 +1536,5 @@
 from openedx.core.djangoapps.plugins import plugin_apps, plugin_settings, constants as plugin_constants
 INSTALLED_APPS.extend(plugin_apps.get_apps(plugin_constants.ProjectType.CMS))
 plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.CMS, plugin_constants.SettingsType.COMMON)
+
+COURSE_EXPORT_DOWNLOAD_CHUNK_SIZE = 4096
```

Comment:
Can you add a short comment above this line to note why you chose the value of 4096? My thinking is that someone unfamiliar with the PR will come into this file and see this constant value but be unaware of why it was chosen instead of a different value.

Reply:
Honestly, I did not have a good reason for picking 4096 other than it was the value used in my source of reference. Also, it worked and had the desired outcome. So I've now set it to the default of ``8192``.